gui-agent/vmside.c

```diff
@@ -876,7 +865,6 @@ static void process_xevent_configure(Ghandles * g, XID window,
     struct msg_hdr hdr;
     struct msg_configure conf;
     struct genlist *l;
-    /* SKIP_NONMANAGED_WINDOW; */
```

Comment:
This one was kind of intentional, as a hint the below fragment is open-coded version of this macro. Maybe too subtle one...